admins/pageflow/entry_templates.rb

```diff
@@ -0,0 +1,66 @@
+module Pageflow
+  ActiveAdmin.register EntryTemplate, as: 'EntryTemplate' do
+    menu false
+    config.batch_actions = false
+    actions :new, :create, :edit, :update
```

Comment:
I still think only providing `edit` and overriding `find_resource` (as mention [here](https://redmine.codevise.de/issues/17306#note-56)) would have been a good idea. Is there an issue with that approach I'm not aware of? Now validations/constraints ensure db consistency, but I can still follow a stale "new" link to a form that I will never be able to save after I fill it in, right?

Reply:
> I'm not aware of? 

Not sure. See [reply to your comment here](https://redmine.codevise.de/issues/17306#note-59).

Basically, I agree with your problem description. See my reply for more on this

Reply:
I'll add that idea to the todo list in any case, so no need to discuss it right now

Reply:
Ah, I missed your comment on the issue. Since discussion moved on over there, I'll reply here.

> Otherwise, opening the edit form twice and saving twice would still create two entry templates for the same account/entry type

I don't think so. All forms always have action urls of the form `/account/5/entry_tempaltes/scrolled` or `/account/5/entry_tempaltes/paged`. The `find_resource` finds or initializes entry templates. Update always just calls save. So the second form submission would be an update either way. The second update would still override attributes with potentially stale data. But that's a problem we have with every admin form if you open multiple instances.

Reply:
Thinking about this some more, I'm not sure just deleting the existing data is an improvement. There is a clear validation error message which can prompt the user to either dismiss stale data, or use the edit link instead. It's definitely annoying to have to copy over the data, but we could also argue that we have some unnecessary duplication of data between entry templates which significantly contributes to this being an annoyance in the first place.

I'd even favor leaving this as it is, and if users request it or if we have time, make entry template entry UX more pleasant by avoiding duplication and, if it then still seems like a nuisance, go with the approach to only have edit/update actions.

Would it be okay with you to leave this as it is in the first iteration?

Reply:
PS: Maybe in this case it would make more sense to label the 'Edit' link for the create action with 'Create', and also let the forms' page titles differ somewhat

Reply:
Sounds good. That way there is also an easy way to see that an account is still missing an (explicitly configured) entry template.

Reply:
Alright! Updated my todos accordingly